### PR TITLE
Installation instructions in README.md

### PR DESCRIPTION
**Pull request by GurshehzadSingh**
# Description

(Write your answer here.)

## Related Issues or Pull Requests

#(Mention Issue Number Here.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by avinashkranjan** [comment]
@GurshehzadSingh Kindly Make the Asked Changes

**Comment by GurshehzadSingh** [comment]
> @GurshehzadSingh Kindly Make the Asked Changes

Sure.. I didn't notice the changes asked.. I'll do it

File: README.md [hunk 1]
```diff
@@ -38,6 +38,51 @@ This module would provide an automated, AI enabled chatbot which would provide i
...
+_**Step by Step Guide to install the project**_ <br/>
+-Fork the repository and create a clone of it on your computer.
+-Go to the the command line and type cd <-name of cloned directory->
+-Run the following command to import the node modules.
+
+```
+npm install
+```
+
+-Run the following command to start the development server.
```
[the remark this answers is not in the document]
Comment (GurshehzadSingh) [reply README.md:56]: Done

File: README.md [hunk 2]
```diff
@@ -38,6 +38,51 @@ This module would provide an automated, AI enabled chatbot which would provide i
...
+
+```
+npm start
+```
+
+_**Installing the backend**_ <br/>
```
[the remark this answers is not in the document]
Comment (GurshehzadSingh) [reply README.md:62]: Done

File: README.md [hunk 3]
```diff
@@ -38,6 +38,51 @@ This module would provide an automated, AI enabled chatbot which would provide i
...
+
+```
+python manage.py migrate
+```
+
+```
+python manage.py runserver
+```
+
+-Your project is running on your local host http://127.0.0.1.
```
[the remark this answers is not in the document]
Comment (GurshehzadSingh) [reply README.md:84]: Done

**Review by GurshehzadSingh — COMMENTED** [review]
I have committed all the changes

**Review by chetan-rns — APPROVED** [review]

**Pull request merged**